templates/repo/settings/options.tmpl

```diff
@@ -81,7 +81,7 @@
 			<div class="ui attached segment">
 				{{if $newMirrorsEntirelyEnabled}}
 					{{$.locale.Tr "repo.settings.mirror_settings.docs"}}
-					<a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/repo-mirror/#pushing-to<ing-to-a-remote-repository">{{$.locale.Tr "repo.settings.mirror_settings.docs.doc_link_title"}}</a><br><br>
+					<a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/usage/repo-mirror/#pushing-to-a-remote-repository">{{$.locale.Tr "repo.settings.mirror_settings.docs.doc_link_title"}}</a><br><br>
```

Comment:
```suggestion
					<a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.com/next/usage/repo-mirror#pushing-to-a-remote-repository">{{$.locale.Tr "repo.settings.mirror_settings.docs.doc_link_title"}}</a><br><br>
```

Reply:
There are 33 matches for `docs.gitea.io/en-us` in templates, I thought someone ought to do this all at once instead of singling out this URL :)

Reply:
I will do that on another PR